src/joycontrolstick.h

```diff
@@ -56,16 +56,16 @@ class JoyControlStick : public QObject, public JoyStickDirectionsType
     void queueJoyEvent(bool ignoresets); // JoyControlStickEvent class
     void activatePendingEvent();         // JoyControlStickEvent class
     void clearPendingEvent();            // JoyControlStickEvent class
-    void setCalibrationFlag(bool flag);
-    void setCalibrationSummary(QString text);
-
-    QString getCalibrationSummary();
 
     bool inDeadZone();
     bool hasSlotsAssigned();
     bool isRelativeSpring();
     bool hasPendingEvent(); // JoyControlStickEvent class
-    bool wasCalibrated();
+
+    bool isCalibrated() const;
+    void resetCalibration();
+    void getCalibration(double *data) const;
```

Comment:
This method is not very elegent.  
Moreover, there is no description, how `data` should be prepared or how it should look like. There is no clear indication what is format(order) of returned values 
Maybe 4 pointers to selected values would be better.

Reply:
I changed it to four pointers. The doxygen description is in the cpp file.